Gemfile

```diff
@@ -44,6 +44,8 @@ gem 'bootsnap', require: false
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/224431269-2db4e0a2-1b20-4468-a8de-1d00927de3c2.png)

**[Optional]**

- [ ] It would be nice if you make your Getting Started section look like below to make your readme descriptive and professional 😊

## Getting Started
### Prerequisites
```
Ruby
IDE (Preferable => Visual Studio Code)
```
### Setup
```
cd my-folder
git clone git@github.com:Ibnballo1/my_blog.git
cd my_blog
```
### Install
```
bundle install
rails db:create
rails db:migrate
```
### Usage
```
rails server or rails s
```
### Run tests
```
rspec spec
```